### DISCUSSION TITLE
Release instead of Tag

### DISCUSSION THREAD

**Discussion by tedt10e** [jeremyevans/roda#235] [General]
Hello Jeremy,

Is it possible to use Release instead of Tag when you release a new version?

Github does not send any notification to the subscriber for a new tag.
It will only send the notification if we use the new release.

Many Thanks 🙏

**Comment by jeremyevans** [comment]
The tag happens at the git level (`git tag`), it's not specific to GitHub.  I don't want to spend the time to create a GitHub release for all new versions, considering that almost everyone will be downloading roda through rubygems.

GitHub releases are designed to support distribution of software (`You can create a release to package software, along with release notes and links to binary files, for other people to use.`).  It's not very useful for Ruby libraries. I do use GitHub releases for software that is primarily downloaded via GitHub, such as for aqualung (https://github.com/jeremyevans/aqualung/releases).

To get notified about new releases, please watch GitHub Discussions, as all new releases will now be announced through GitHub Discussions.

**Comment by milouse** [comment]
If necessary, there is also an atom feed for the tags you can subscribe to: https://github.com/jeremyevans/roda/tags.atom
